dotcom-rendering/makefile

```diff
@@ -109,9 +109,9 @@ cypress-open: clear clean-dist install build
 	$(call log, "starting frontend PROD server and opening Cypress")
 	@NODE_ENV=production DISABLE_LOGGING_AND_METRICS=true start-server-and-test 'node dist/frontend.server.js' 9000 'cypress open --e2e --browser electron'
 
-ampValidation: clean-dist install build
-	$(call log, "starting frontend PROD server for AMP Validation")
-	@NODE_ENV=production DISABLE_LOGGING_AND_METRICS=true start-server-and-test 'node dist/frontend.server.js' 9000 'node scripts/test/amp-validation.js'
+ampValidation: clean-dist install
+	$(call log, "starting AMP Validation test")
+	@node scripts/test/amp-validation.js
```

Comment:
This probably isn't so applicable in this action, since we rarely run it locally - but for some others we might want to consider having ci & dev versions of commands so that when running in dev mode the server, etc is taken care of for you!